### PR TITLE
Use std::fabs() instead of abs() in rationalTime.cpp

### PR DESCRIPTION
**Pull request by KarthikRIyer**

### CODE REVIEW

File: src/opentime/rationalTime.cpp [hunk 1]
```diff
@@ -290,7 +290,7 @@ RationalTime::to_time_string() const {
     // result and return the string at the end with a '-'. This provides
     // compatibility with ffmpeg, which allows negative time strings.
     if (std::signbit(total_seconds)) {
-        total_seconds = abs(total_seconds);
+        total_seconds = std::abs(total_seconds);
```
[the remark this answers is not in the document]
Comment (meshula) [reply src/opentime/rationalTime.cpp:293]: In this local context, abs doesn't cause any bugs through integral promotion. My recommendation to correct the c standard library abs to std::abs was in error, it should be std::fabs as you say. We are idiomatically using the std:: wrapped C functions throughout otio. Under the hood, they should evaluate to the same thing.

[the remark this answers is not in the document]
Comment (ssteinbach) [reply src/opentime/rationalTime.cpp:293]: ok cool.  So then just `std::fabs` change as noted.  Thanks!  Good catch!

**Comment by meshula** [comment]
@ssteinbach @reinecke The tox build failed - is that spurious, or related to this change in some way?

**Comment by ssteinbach** [comment]
Looks like the run failed just for python3, while trying to run coverage.  I'll restart it.

**Pull request merged**